src/app/component/CartItem/CartItem.component.js

```diff
@@ -274,17 +282,41 @@ export class CartItem extends PureComponent {
             item: {
                 customizable_options,
                 bundle_options
-            } = {}
+            } = {},
+            device
         } = this.props;
 
+        if (device.isMobile) {
+            return (
+                <figcaption
+                  block="CartItem"
+                  elem="Content"
+                  mods={ { isLikeTable } }
+                >
+                    { this.renderOutOfStockMessage() }
+                    <div block="CartItem" elem="HeadingWrapper">
+                        { this.renderProductName() }
+                        { this.renderDeleteButton() }
+                    </div>
+                    { this.renderProductOptions(customizable_options) }
+                    { this.renderProductOptions(bundle_options) }
+                    { this.renderProductConfigurations() }
+                    { this.renderQuantityChangeField() }
+                    { this.renderProductPrice() }
+                </figcaption>
+            );
+        }
+
         return (
             <figcaption
               block="CartItem"
               elem="Content"
               mods={ { isLikeTable } }
             >
                 { this.renderOutOfStockMessage() }
-                { this.renderProductName() }
+                <div block="CartItem" elem="HeadingWrapper">
+                    { this.renderProductName() }
+                </div>
                 { this.renderProductOptions(customizable_options) }
                 { this.renderProductOptions(bundle_options) }
                 { this.renderProductConfigurations() }
```

Comment:
Both returns have almost identical results.
Could you please move code parts that differ to a separate method and do the browser check there?
This method should have exactly one return in this case.